python/manageJobs.py

```diff
@@ -260,7 +280,39 @@ def manageJobs(argv=None):
                 # actions that can be applied to all jobs
                 jobnums = [j.num for j in jobs]
                 schedd.act(htcondor.JobAction.Remove,jobnums)
+            elif options.xrootdResubmit:
+                print ""
+                file_and_site_per_file = find_input_file_site_per_job(argv = (["-c", options.inputFileClassAd] if options.inputFileClassAd else []) +

```

Comment:
please structure argument indentations like this:
```python
foo = bar(
    lorem = (
        ipsum,
    )
)
```

Reply:
Done